src/kOS/Module/kOSProcessor.cs

```diff
@@ -82,7 +95,7 @@ public void Activate()
             OpenWindow();
         }
 
-        [KSPField(isPersistant = true, guiName = "Required Power", guiActive = true)]
+        [KSPField(isPersistant = true, guiName = "kOS Required Power", guiActive = true, guiActiveEditor = true)]
```

Comment:
Maybe 'kOS Peak Consumption"?

Be careful about rounding the value that it doesn't round down to zero in cases where the value is very small and people have reduced the power setting a lot, like RO seems to be planning to do.  Maybe round it by number of significant digits rather than by a fixed decimal place, so when the number is small it gets more places to the right of the decimal point.

It occurs to me it may be useful for us to maintain a memory of how much power got used during the last few fixedupdates, and say every 10 or 20 fixedupdates, get the average and display it in the context menu alongside the max value.  That's good enough for visual display and yet it smooths out the bouncy number problem you're talking about.

(And maybe we do actually want to expose the most recent power drain value to scripts too, as a suffix of `core`.)